app/build.gradle.kts

```diff
@@ -124,7 +124,7 @@ android {
             resValue("string", "app_name", "Brainwallet (screengrab)")
             buildConfigField("String[]", "SCREENGRAB_PAPERKEY", 
                 "new String[] {${localProperties.getProperty("SCREENGRAB_PAPERKEY", "")
-                    .split(",")
+                    .split(" ")
```

Comment:
if this merged, please replace the comma to space `SCREENGRAB_PAPERKEY` inside `local.properties`

Reply:
got it